### ISSUE TITLE
Opening rrd file with same application id as current one does not work

### ISSUE THREAD

**Issue by roym899** [rerun-io/rerun#3987]
**Describe the bug**
When trying to load a different rrd file with the same application_id as the current one, nothing happens. (both by drag and drop, and by using the load dialog). 

Opening from the terminal starts a separate Rerun instance, which I think is good.

**To Reproduce**
Create two rrd files
```python
import rerun as rr
import numpy as np
import time

rr.init("test", spawn=True)

rr.save("test1.rrd")
rr.log("image", rr.Image(0.3 * np.random.rand(100,100,3)))

time.sleep(5.0)

rr.save("test2.rrd")
rr.log("image", rr.Image(np.random.rand(100,100,3)))
```
Drag and drop one of the two files into the viewer (rrd file shows up). 
Drag and drop the other file into the viewer (-> nothing changes).

**Expected behavior**
Open the new rrd file. Showing both in the list of recent recordings.

**Rerun version**
```
rerun_py 0.10.0-alpha.7+dev [rustc 1.72.1 (d5c2e9c34 2023-09-13), LLVM 16.0.5] x86_64-unknown-linux-gnu main 47414c8, built 2023-10-24T12:00:38Z
```

**Comment by nikolausWest** [comment]
This is probably due to:
- https://github.com/rerun-io/rerun/issues/3793

tldr: Both rrd files actually have the same application id and recording id because in Python, recording id is set to authkey() by default which is random but constant per process
